docs/blog/2018-10-09-uptick-docs-contributions-hacktoberfest/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Uptick in docs contributions for Gatsby Hacktoberfest
+date: 2018-10-09
+author: Shannon Soper
+tags: ["documentation", "hacktoberfest"]
+---
+
+## Hacktoberfest docs explosion!
+
+Since October 1st -- the start of Hacktoberfest -- the Gatsby OSS community has seen a huge uptick in docs contributions. In the first 5 days of October we [merged 28 PR's with the `documentation` label](https://github.com/gatsbyjs/gatsby/pulls?utf8=%E2%9C%93&q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+label%3A%22type%3A+documentation%22+language%3Aswift+closed%3A%3E2018-10-01) (and by the time you click on that link, it will likely be more than 28, especially if PR’s are labelled correctly!).
+
+A big thanks to every contributor who jumped in to help Gatsby docs keep becoming more polished, comprehensive, and clear. In fact, many of the PR’s fixed problems we hadn’t even created issues for yet! And we’re increasing the number of `hacktoberfest` + `documentation` labelled issues to account for the surge of helpful contributions. A special thanks to [@amberleyromo](https://github.com/amberleyromo), [@jlengstorf](https://github.com/jlengstorf), [@DSchau](https://github.com/DSchau), and [@pieh](https://github.com/pieh) for their hard work giving feedback and merging PR’s!
+
+Read more on how to participate in Gatsby Hacktoberfest in our [Hacktoberfest kickoff post](/blog/2018-10-09-hacktoberfest-kickoff/)
+
+## With great contributions comes great responsibility
+
+The increase in contributions means the Gatsby community makes more frequent decisions about what goes in the docs and what doesn't, how to title them, the best slugs for SEO, and when to create new categories.
+
+This has led to two changes:
+
+1.  [@jlengstorf](https://github.com/jlengstorf) created the [@gatsbyjs/docs team](https://github.com/orgs/gatsbyjs/teams/docs) in GitHub that automatically notifies whomever is on the team whenever a PR includes edits to anything within /docs/.
+
+2.  With the help of many people, I created a “docs decision tree,” so that when the @docsteam is notified, they can make decisions in a unified manner according to a set of agreed-upon principles (which can be adjusted over time). This will help anyone making decisions pertaining to docs escape any biases and fickleness. The “docs decision tree” will be tested over Hacktoberfest with all issues labelled `documentation` and collect discussion in the [Docs Decision Tree RFC](https://github.com/gatsbyjs/rfcs/pull/14).
```

Comment:
maybe change "@docsteam" to "@gatsbyjs/docs" so it's consistent?

Reply:
Yep -- I did this elsewhere, I agree